landoapi/cli.py

```diff
@@ -167,7 +162,16 @@ def lint():
     os.execvp("flake8", ("flake8",) + LINT_PATHS)
 
 
-@cli.command(with_appcontext=False, context_settings=dict(ignore_unknown_options=True))
+@cli.command(with_appcontext=False)
```

Comment:
The `lint` command can be removed, or maybe we move the contents of `ruff` into it?